pom.xml

```diff
@@ -1188,6 +1188,8 @@
 
     <module>arquillian</module>
 
+    <module>fractions/health-api</module>
```

Comment:
i don't see the problem with also moving the other module, but if you think its a bad idea that's fine